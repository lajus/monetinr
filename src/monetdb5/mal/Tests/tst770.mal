function test(b:bat[:any_1,:any_2],c:any_2):void;
        reduce := batcalc./(b,c); 
end test;
function test(b:bat[:any_1,:any_2],c:any_1):void;
        reduce := batcalc./(b,c); 
end test;
function test(b:bat[:any_1,:any_2]):void;
        reduce := batcalc./(b,1); 
end test;
