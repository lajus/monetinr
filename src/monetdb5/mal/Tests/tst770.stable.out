stdout of test 'tst770` in directory 'monetdb5/mal` itself:


# 08:01:31 >  
# 08:01:31 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/Linux/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB5/log" --set mapi_port=42158 --set sql_port=55708 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst770.mal </dev/null
# 08:01:31 >  

# Monet Database Server V4.99.19
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# config:/ufs/mk/monet5/Linux/etc/MonetDB5.conf
# dbfarm:/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
#function test(b:bat[:any_1,:any_2],c:any_2):void;
#        reduce := batcalc./(b,c); 
#end test;
function user.test(b:bat[:any_1,:any_2],c:any_2):void;
    reduce := batcalc./(b,c);
end test;
#function test(b:bat[:any_1,:any_2],c:any_1):void;
#        reduce := batcalc./(b,c); 
#end test;
function user.test(b:bat[:any_1,:any_2],c:any_1):void;
    reduce := batcalc./(b,c);
end test;
#function test(b:bat[:any_1,:any_2]):void;
#        reduce := batcalc./(b,1); 
#end test;
function user.test(b:bat[:any_1,:any_2]):void;
    reduce := batcalc./(b,1);
end test;
#

# 08:01:31 >  
# 08:01:31 >  Done.
# 08:01:31 >  

